VotingApplication/VotingApplication.Web/Content/Scss/Voting.scss

```diff
@@ -172,22 +184,17 @@ hr {
 
 .option-buttons > * {
     display: block;
-    text-align:center;
-    width: 100%
+    text-align: center;
+    width: 100%;
 }
 
-.btn-group {
-    border: 1px solid $Green700;
-    border-radius: 5px;
-    overflow:hidden;
-    margin-bottom: 5px;
+.btn-group-points {
```

Comment:
Something weird seems to be happening to the points buttons now. They don't line up like they used to.

It looks like they are 102px across when on one line, but being squashed into 100px